Makefile

```diff
@@ -353,6 +354,7 @@ test-e2e: test-e2e-setup
 	--ginkgo.label-filter="$(TEST_FILTER)" \
 	-ci_cred_file=$(CI_CRED_FILE) \
 	-provider=$(CLUSTER_TYPE) \
+	-stream=$(OADP_STREAM) \
```

Comment:
This was not in https://github.com/openshift/oadp-operator/pull/811/
Do you want to put this change in master?
https://github.com/openshift/oadp-operator/blob/master/Makefile